WordPress/Classes/ViewRelated/Reader/ReaderBrowseSiteViewController.h

```diff
@@ -2,5 +2,10 @@
 
 @class ReaderPost;
 @interface ReaderBrowseSiteViewController : UIViewController
+
 - (instancetype)initWithPost:(ReaderPost *)post;
+
+- (instancetype)initWithSiteID:(NSNumber *)siteID
+                       siteURL:(NSString *)siteURL
+                       isWPcom:(BOOL)isWPcom;
```

Comment:
By the way sir, thank you for this method!. I'll be integrating Notifications with `ReaderBrowseSiteViewController`!